### PR TITLE
Javascriptbranch

### PR DESCRIPTION
**Pull request by KlausR98**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1 +1,63 @@
-console.log("commit 1");
+const coin = document.querySelector(".coin");
+const flipButton = document.querySelector("#flip-button");
+const resetButton = document.querySelector("#reset-button");
+let heads = 0;
+let tails = 0;
+
+// flipButton.addEventListener("click", () => flipCoin(coin, heads, tails));
```
Comment (jirayasensei) [inline script.js]: should it be here? if not, delete please.

File: script.js [hunk 2]
```diff
@@ -1 +1,63 @@
...
+//     heads++;
+//   } else {
+//     setTimeout(function () {
+//       coin.style.animation = "tail-spin 3s forwards";
+//     }, 100);
+//     tails++;
+//   }
+//   setTimeout(changeScore, 3000);
+//   turnOffFlipButton();
+// }
```
Comment (jirayasensei) [inline script.js]: same as above

File: script.js [hunk 3]
```diff
@@ -1 +1,63 @@
...
+
+flipButton.addEventListener("click", () => {
+  let i = Math.floor(Math.random() * 2);
+  coin.style.animation = "none";
+  // if (heads++) {
+
+  // }
```
Comment (jirayasensei) [inline script.js]: same

File: style.css [hunk 4]
```diff
@@ -71,3 +96,23 @@ body {
...
+
+@keyframes head-spin {
+  from {
+    transform: rotateX(0);
+  }
+
+  to {
+    transform: rotateX(2160deg);
+  }
+}
```
Comment (jirayasensei) [inline style.css:118]: very nice implementation, good work

File: script.js (hunk 1 above)
Comment (KlausR98) [reply script.js]: Deleted

File: script.js (hunk 2 above)
Comment (KlausR98) [reply script.js]: Deleted

File: script.js (hunk 3 above)
Comment (KlausR98) [reply script.js]: Deleted

**Pull request merged**